R/asdataframe.R

```diff
@@ -0,0 +1,54 @@
+#' as.data.frame method for secuTrialdata objects
+#'
+#' @param object secuTrialdata object
+#' @param data.frames character vector of data.frame names to turn into data.frames
+#' @param meta logical should metadata be returned
+#' @param regex regex syntax to remove from names
+#' @param rep replacement for regex
+#' @param envir environment in which to put the data
+#'
+#' @return each data.frame on the secuTrialdata object is saved to it's own data.frame in the environment
+#' @export
+#'
+#' @examples
+#' # prepare path to example export
+#' export_location <- system.file("extdata",
+#'                                "s_export_CSV-xls_CTU05_longnames_sep_ref.zip",
+#'                                package = "secuTrialR")
+#' # load all export data
+#' sT_export <- read_secuTrial_export(data_dir = export_location)
+#' # add files to global environment
+#' as.data.frame(sT_export)
+#' # add files to global environment, removing the project name from the file names
+#' as.data.frame(sT_export, regex = "ctu05")
+as.data.frame.secuTrialdata <- function(object,
+                                        data.frames = NULL,
+                                        meta = TRUE,
```

Comment:
Naively I was thinking `meta = FALSE` would be better. What is your reason to set the default to `TRUE`?

Reply:
Thats a fair point. No particular reason at all actually...